packages/util/src/bn/index.ts

```diff
@@ -6,6 +6,7 @@
  * @summary Utility methods to convert to and from `BN` objects
  */
 
+export * from './consts';
```

Comment:
Not sure about this since it is not user the BN_* namespace. So if the consts were `BN_<SOMETHING>` it would work.